lib/src/kvs/ds.rs

```diff
@@ -821,25 +848,132 @@ impl Datastore {
 	// It is handy for testing, because it allows you to specify the timestamp,
 	// without depending on a system clock.
 	pub async fn tick_at(&self, ts: u64) -> Result<(), Error> {
-		self.save_timestamp_for_versionstamp(ts).await?;
+		let vs = self.save_timestamp_for_versionstamp(ts).await?;
+		if FFLAGS.change_feed_live_queries.enabled() {
+			self.process_lq_notifications(ts, vs).await?;
```

Comment:
If we have too many live queries to process, this will delay subsequent ticks. I'm not sure if this is intentional? /cc @mumoshu

Reply:
Oh yes, good point! Will think about that because that is a problem

Reply:
Changed to spawn (multi-threaded) and conditional compile for wasm to use bindgen